modules/oracle-xe/src/main/java/org/testcontainers/containers/OracleContainer.java

```diff
@@ -108,4 +108,9 @@ public Integer getWebPort() {
     public String getTestQueryString() {
         return "SELECT 1 FROM DUAL";
     }
+
+    @Override
+    protected void waitUntilContainerStarted() {
```

Comment:
why do you need to override it?

Reply:
I checked PostgreSQLContainer and it overrides waitUntilContainerStarted method but after your comment i also checked MySQLContainer which does not. I suppose that WaitStrategy for most JdbcDatabaseContainer containers is the same.

Reply:
PostgreSQLContainer uses log-based wait strategy, while others inherit the JDBC one, hence the override

Reply:
fixed